docs/config/lua/config/font_rules.md

```diff
@@ -45,48 +48,56 @@ return {
 }
 ```
 
-Here's an example from my configuration file;
+Here's an example from a configuration file;
 
 ```lua
 local wezterm = require 'wezterm'
 
 -- A helper function for my fallback fonts
 function font_with_fallback(name, params)
-  local names = { name, 'Noto Color Emoji', 'JetBrains Mono' }
+  local names = { name, 'Noto Color Emoji', 'Symbols Nerd Font Mono' }
   return wezterm.font_with_fallback(names, params)
 end
 
 return {
-  font = font_with_fallback 'Operator Mono SSm Lig Medium',
+  font = font_with_fallback('FiraCode', {weight = 'Light'}),
+  font_size = 8.0,
```

Comment:
Let's take `font_size` out of this page: this value is much much smaller than the default font size, and will harm readability/accessibility for a number of users. Additionally, `font_size` is an independent concept from `font_rules`, so it doesn't need to be in this example.